src/components/modules/header/Header.tsx

```diff
@@ -83,145 +46,56 @@ const subMessageStyles = css`
     margin: 5px 0;
 `;
 
-export enum HeaderVariant {
-    Control,
-    SupportAgain,
-}
-
-export const getHeader: (variant: HeaderVariant) => React.FC<HeaderProps> = (
-    variant: HeaderVariant,
-) => (props: HeaderProps) => {
+const _Header: React.FC<HeaderRenderProps> = (props: HeaderRenderProps) => {
```

Comment:
is the `_` a convention in react naming?

Reply:
I think this was more from a general `_` = `local/private` naming convention from a few different languages (e.g `python`). Instead of 

```ts
const _Header = /* ... */
export const Header = headerWrapper(Header)
```

we could do

```ts
const Header = /* ... */
const wrapped = headerWrapper(Header)

export { wrapped as Header }
```

what do you reckon?

Reply:
Ok, I think whichever we choose it should be consistent within this codebase. In the existing code there are examples of the 2nd style, though I'm not averse to adopting `_` instead

Reply:
Yeah makes sense - it's probably simplest for now that we just adopt the second style for consistency